docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/cthp/node--cgov-cthp--full.html.twig

```diff
@@ -0,0 +1,29 @@
+{# Page Content #}
+{% set isPatient = (content.field_audience|field_value|render|render is same as('patient')) %}
+{% set displayLang = content.field_audience["#language"] %}
+{% if content.field_audience_toggle[0] %}
+  {% set showToggle = true %}
+  {% set url = path('entity.node.canonical', {'node':  content.field_audience_toggle[0]["#node"].nid[0].value}) %}
+{% endif %}
+{% set toggleText = (isPatient ? 'Go to Health Professional Version' : 'Go to Patient Version')|trans %}
```

Comment:
`|trans` ?

Reply:
I will forget this as part of https://github.com/NCIOCPL/cgov-digital-platform/issues/3384